tests/data/SD_ME0WME0W/dataset_ingest_config.yml

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
SD_ME0WME0W study is used here to test the docker build during the build stage of CI [here](https://github.com/kids-first/kf-lib-data-ingest-config/blob/e7b893633ce3176260de7e6a27215caacc75e233/aws-ecs-service-type-1/ci-scripts/build_stage/build.sh#L22)